src/sas/sascalc/dataloader/data_info.py

```diff
@@ -92,14 +39,17 @@ class plottable_1D(object):
     dxl = None
     ## Slit smearing width
     dxw = None
+    ## SESANS specific params (wavelengths for spin echo length calculation)
+    lam = None
+    dlam = None

```

Comment:
Not used anywhere...yet. 
It will be needed for acceptance angle limits implementation. 
If it's a real problem I will remove it, otherwise I request to leave it in.